### ISSUE TITLE
Cannot load package

### ISSUE THREAD

**Issue by EvoArt** [arviz-devs/ArviZ.jl#206]
Hi there

On a windows laptop, using Julia 1.7.2 in a clean env, I get the following

```julia
julia> using Pkg

julia> Pkg.activate(".")
  Activating new project at `C:\Users\arn203\OneDrive - University of Exeter\Documents\testenv`

julia> Pkg.add("ArviZ")
    Updating registry at `C:\Users\arn203\.julia\registries\General`
    Updating git-repo `https://github.com/JuliaRegistries/General`
   Resolving package versions...
   Installed DocStringExtensions ─ v0.9.1
    Updating `C:\Users\arn203\OneDrive - University of Exeter\Documents\testenv\Project.toml` 
  [131c737c] + ArviZ v0.6.0
    Updating `C:\Users\arn203\OneDrive - University of Exeter\Documents\testenv\Manifest.toml`
  [79e6a3ab] + Adapt v3.3.3
  [4fba245c] + ArrayInterface v6.0.21
  [30b0a656] + ArrayInterfaceCore v0.1.15
  [131c737c] + ArviZ v0.6.0
  [d360d2e6] + ChainRulesCore v1.15.3
  [9e997f8a] + ChangesOfVariables v0.1.4
  [3da002f7] + ColorTypes v0.11.4
  [5ae59095] + Colors v0.12.8
  [34da2185] + Compat v3.45.0
  [8f4d0f93] + Conda v1.7.0
  [187b0558] + ConstructionBase v1.4.0
  [a8cc5b0e] + Crayons v4.1.1
  [9a962f9c] + DataAPI v1.10.0
  [a93c6f00] + DataFrames v1.3.4
  [864edb3b] + DataStructures v0.18.13
  [e2d170a0] + DataValueInterfaces v1.0.0
  [0703355e] + DimensionalData v0.20.11
  [ffbed154] + DocStringExtensions v0.9.1
  [411431e0] + Extents v0.1.1
  [53c48c17] + FixedPointNumbers v0.8.4
  [59287772] + Formatting v0.4.2
  [615f187c] + IfElse v0.1.1
  [8197267c] + IntervalSets v0.7.1
  [3587e190] + InverseFunctions v0.1.7
  [41ab1584] + InvertedIndices v1.1.0
  [92d709cd] + IrrationalConstants v0.1.1
  [82899510] + IteratorInterfaceExtensions v1.0.0
  [682c06a0] + JSON v0.21.3
  [b964fa9f] + LaTeXStrings v1.3.0
  [2ab3a3ac] + LogExpFunctions v0.3.16
  [1914dd2f] + MacroTools v0.5.9
  [e1d29d7a] + Missings v1.0.2
  [bac558e1] + OrderedCollections v1.4.1
  [ce719bf2] + PSIS v0.5.1
  [69de0a69] + Parsers v2.3.2
  [2dfb63ee] + PooledArrays v1.4.2
  [08abe8d2] + PrettyTables v1.3.1
  [438e738f] + PyCall v1.93.1
  [d330b81b] + PyPlot v2.10.0
  [3cdcf5f2] + RecipesBase v1.2.1
  [189a3867] + Reexport v1.2.2
  [ae029012] + Requires v1.3.0
  [a2af1166] + SortingAlgorithms v1.0.1
  [aedffcd0] + Static v0.7.6
  [82ae8749] + StatsAPI v1.4.0
  [2913bbd2] + StatsBase v0.33.20
  [3783bdb8] + TableTraits v1.0.1
  [bd369af6] + Tables v1.7.0
  [81def892] + VersionParsing v1.3.0
  [0dad84c5] + ArgTools
  [56f22d72] + Artifacts
  [2a0f44e3] + Base64
  [ade2ca70] + Dates
  [8bb1440f] + DelimitedFiles
  [8ba89e20] + Distributed
  [f43a241f] + Downloads
  [9fa8497b] + Future
  [b77e0a4c] + InteractiveUtils
  [b27032c2] + LibCURL
  [76f85450] + LibGit2
  [8f399da3] + Libdl
  [37e2e46d] + LinearAlgebra
  [56ddb016] + Logging
  [d6f4376e] + Markdown
  [a63ad114] + Mmap
  [ca575930] + NetworkOptions
  [44cfe95a] + Pkg
  [de0858da] + Printf
  [3fa0cd96] + REPL
  [9a3f8284] + Random
  [ea8e919c] + SHA
  [9e88b42a] + Serialization
  [1a1011a3] + SharedArrays
  [6462fe0b] + Sockets
  [2f01184e] + SparseArrays
  [10745b16] + Statistics
  [4607b0f0] + SuiteSparse
  [fa267f1f] + TOML
  [a4e569a6] + Tar
  [8dfed614] + Test
  [cf7118a7] + UUIDs
  [4ec0a83e] + Unicode
  [e66e0078] + CompilerSupportLibraries_jll
  [deac9b47] + LibCURL_jll
  [29816b5a] + LibSSH2_jll
  [c8ffd9c3] + MbedTLS_jll
  [14a3606d] + MozillaCACerts_jll
  [4536629a] + OpenBLAS_jll
  [83775a58] + Zlib_jll
  [8e850b90] + libblastrampoline_jll
  [8e850ede] + nghttp2_jll
  [3f19e933] + p7zip_jll
Precompiling project...
  ✗ ArviZ
  4 dependencies successfully precompiled in 104 seconds (50 already precompiled)
  1 dependency errored. To see a full report either run `import Pkg; Pkg.precompile()` or load the package

julia> Pkg.status()
      Status `C:\Users\arn203\OneDrive - University of Exeter\Documents\testenv\Project.toml`
  [131c737c] ArviZ v0.6.0

julia> using ArviZ
[ Info: Precompiling ArviZ [131c737c-5715-5e2e-ad31-c244f01c1dc7]
[ Info: Installing arviz via the Conda arviz package...
[ Info: Running `conda config --add channels conda-forge --file 'C:\Users\arn203\.julia\conda\3\condarc-julia.yml' --force` in root environment
Warning: 'conda-forge' already in 'channels' list, moving to the top
[ Info: Running `conda install -y arviz` in root environment
Collecting package metadata (current_repodata.json): done
Solving environment: done

# All requested packages already installed.

Try installing arviz using pip? [Y/n]: Requirement already satisfied: arviz in c:\users\arn203\.julia\conda\3\lib\site-packages (0.12.1)
Requirement already satisfied: typing-extensions>=3.7.4.3 in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (3.7.4.3)
Requirement already satisfied: netcdf4 in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (1.5.6)
Requirement already satisfied: setuptools>=38.4 in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (52.0.0.post20210125)
Requirement already satisfied: matplotlib>=3.0 in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (3.3.4)      
Requirement already satisfied: packaging in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (20.9)
Requirement already satisfied: xarray-einstats>=0.2 in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (0.3.0) 
Requirement already satisfied: xarray>=0.16.1 in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (0.16.2)
Requirement already satisfied: pandas>=0.23 in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (1.2.2)
Requirement already satisfied: scipy>=0.19 in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (1.7.3)
Requirement already satisfied: numpy>=1.12 in c:\users\arn203\.julia\conda\3\lib\site-packages (from arviz) (1.19.2)
Requirement already satisfied: python-dateutil>=2.1 in c:\users\arn203\.julia\conda\3\lib\site-packages (from matplotlib>=3.0->arviz) (2.8.1)
Requirement already satisfied: pillow>=6.2.0 in c:\users\arn203\.julia\conda\3\lib\site-packages (from matplotlib>=3.0->arviz) (8.1.0)
Requirement already satisfied: cycler>=0.10 in c:\users\arn203\.julia\conda\3\lib\site-packages (from matplotlib>=3.0->arviz) (0.10.0)
Requirement already satisfied: kiwisolver>=1.0.1 in c:\users\arn203\.julia\conda\3\lib\site-packages (from matplotlib>=3.0->arviz) (1.3.1)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.3 in c:\users\arn203\.julia\conda\3\lib\site-packages (from matplotlib>=3.0->arviz) (2.4.7)
Requirement already satisfied: six in c:\users\arn203\.julia\conda\3\lib\site-packages (from cycler>=0.10->matplotlib>=3.0->arviz) (1.15.0)
Requirement already satisfied: pytz>=2017.3 in c:\users\arn203\.julia\conda\3\lib\site-packages (from pandas>=0.23->arviz) (2021.1)
Requirement already satisfied: cftime in c:\users\arn203\.julia\conda\3\lib\site-packages (from netcdf4->arviz) (1.4.1)
ERROR: LoadError: PyError (PyImport_ImportModule) <class 'FileNotFoundError'>
FileNotFoundError("Could not find module 'C:\\Users\\arn203\\.julia\\conda\\3\\lib\\site-packages\\scipy\\.libs\\libbanded5x.CBCBXIAYFUWOYCSBXUA2BT6W76FYX4C7.gfortran-win_amd64.dll' (or one of its dependencies). Try using the full path with constructor syntax.")
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\arviz\__init__.py", line 32, in <module>
    from .data import *
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\arviz\data\__init__.py", line 2, in <module>
    from .base import CoordSpec, DimSpec, dict_to_dataset, numpy_to_data_array
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\arviz\data\base.py", line 11, in <module>
    import xarray as xr
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\__init__.py", line 3, in <module>
    from . import testing, tutorial, ufuncs
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\testing.py", line 8, in <module>
    from xarray.core import duck_array_ops, formatting, utils
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\core\duck_array_ops.py", line 16, in <module>
    from . import dask_array_compat, dask_array_ops, dtypes, npcompat, nputils
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\core\dask_array_compat.py", line 8, in <module>
    import dask.array as da
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\__init__.py", line 2, in <module>
    from .blockwise import blockwise, atop
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\blockwise.py", line 288, in <module>
    from .core import new_da_object
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\core.py", line 69, in <module>
    from .chunk_types import is_valid_array_chunk, is_valid_chunk_type
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\chunk_types.py", line 130, in <module>
    import scipy.sparse
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\__init__.py", line 136, in <module>
    from . import _distributor_init
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\_distributor_init.py", line 59, in <module>
    WinDLL(os.path.abspath(filename))
  File "C:\Users\arn203\.julia\conda\3\lib\ctypes\__init__.py", line 381, in __init__
    self._handle = _dlopen(self._name, mode)

Stacktrace:
  [1] pyimport(name::String)
    @ PyCall C:\Users\arn203\.julia\packages\PyCall\7a7w0\src\PyCall.jl:550
  [2] _import_dependency(modulename::String, pkgname::String; channel::String)
    @ ArviZ C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\setup.jl:129
  [3] import_arviz()
    @ ArviZ C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\setup.jl:1
  [4] top-level scope
    @ C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\ArviZ.jl:113
  [5] include
    @ .\Base.jl:418 [inlined]
  [6] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)
    @ Base .\loading.jl:1318
  [7] top-level scope
    @ none:1
  [8] eval
    @ .\boot.jl:373 [inlined]
  [9] eval(x::Expr)
    @ Base.MainInclude .\client.jl:453
 [10] top-level scope
    @ none:1
in expression starting at C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\ArviZ.jl:2

caused by: PyError (PyImport_ImportModule) <class 'FileNotFoundError'>
FileNotFoundError("Could not find module 'C:\\Users\\arn203\\.julia\\conda\\3\\lib\\site-packages\\scipy\\.libs\\libbanded5x.CBCBXIAYFUWOYCSBXUA2BT6W76FYX4C7.gfortran-win_amd64.dll' (or one of its dependencies). Try using the full path with constructor syntax.")
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\arviz\__init__.py", line 32, in <module>
    from .data import *
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\arviz\data\__init__.py", line 2, in <module>
    from .base import CoordSpec, DimSpec, dict_to_dataset, numpy_to_data_array
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\arviz\data\base.py", line 11, in <module>
    import xarray as xr
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\__init__.py", line 3, in <module>
    from . import testing, tutorial, ufuncs
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\testing.py", line 8, in <module>
    from xarray.core import duck_array_ops, formatting, utils
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\core\duck_array_ops.py", line 16, in <module>
    from . import dask_array_compat, dask_array_ops, dtypes, npcompat, nputils
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\core\dask_array_compat.py", line 8, in <module>
    import dask.array as da
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\__init__.py", line 2, in <module>
    from .blockwise import blockwise, atop
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\blockwise.py", line 288, in <module>
    from .core import new_da_object
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\core.py", line 69, in <module>
    from .chunk_types import is_valid_array_chunk, is_valid_chunk_type
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\chunk_types.py", line 130, in <module>
    import scipy.sparse
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\__init__.py", line 136, in <module>
    from . import _distributor_init
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\_distributor_init.py", line 59, in <module>
    WinDLL(os.path.abspath(filename))
  File "C:\Users\arn203\.julia\conda\3\lib\ctypes\__init__.py", line 381, in __init__
    self._handle = _dlopen(self._name, mode)

Stacktrace:
  [1] pyimport(name::String)
    @ PyCall C:\Users\arn203\.julia\packages\PyCall\7a7w0\src\PyCall.jl:550
  [2] pyimport_conda(modulename::String, condapkg::String, channel::String)
    @ PyCall C:\Users\arn203\.julia\packages\PyCall\7a7w0\src\PyCall.jl:714
  [3] _import_dependency(modulename::String, pkgname::String; channel::String)
    @ ArviZ C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\setup.jl:123
  [4] import_arviz()
    @ ArviZ C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\setup.jl:1
  [5] top-level scope
    @ C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\ArviZ.jl:113
  [6] include
    @ .\Base.jl:418 [inlined]
  [7] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)
    @ Base .\loading.jl:1318
  [8] top-level scope
    @ none:1
  [9] eval
    @ .\boot.jl:373 [inlined]
 [10] eval(x::Expr)
    @ Base.MainInclude .\client.jl:453
 [11] top-level scope
    @ none:1

caused by: PyError (PyImport_ImportModule) <class 'FileNotFoundError'>
FileNotFoundError("Could not find module 'C:\\Users\\arn203\\.julia\\conda\\3\\lib\\site-packages\\scipy\\.libs\\libbanded5x.CBCBXIAYFUWOYCSBXUA2BT6W76FYX4C7.gfortran-win_amd64.dll' (or one of its dependencies). Try using the full path with constructor syntax.")
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\arviz\__init__.py", line 32, in <module>
    from .data import *
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\arviz\data\__init__.py", line 2, in <module>
    from .base import CoordSpec, DimSpec, dict_to_dataset, numpy_to_data_array
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\arviz\data\base.py", line 11, in <module>
    import xarray as xr
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\__init__.py", line 3, in <module>
    from . import testing, tutorial, ufuncs
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\testing.py", line 8, in <module>
    from xarray.core import duck_array_ops, formatting, utils
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\core\duck_array_ops.py", line 16, in <module>
    from . import dask_array_compat, dask_array_ops, dtypes, npcompat, nputils
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\xarray\core\dask_array_compat.py", line 8, in <module>
    import dask.array as da
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\__init__.py", line 2, in <module>
    from .blockwise import blockwise, atop
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\blockwise.py", line 288, in <module>
    from .core import new_da_object
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\core.py", line 69, in <module>
    from .chunk_types import is_valid_array_chunk, is_valid_chunk_type
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\dask\array\chunk_types.py", line 130, in <module>
    import scipy.sparse
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\__init__.py", line 136, in <module>
    from . import _distributor_init
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\_distributor_init.py", line 59, in <module>
    WinDLL(os.path.abspath(filename))
  File "C:\Users\arn203\.julia\conda\3\lib\ctypes\__init__.py", line 381, in __init__
    self._handle = _dlopen(self._name, mode)

Stacktrace:
  [1] pyimport(name::String)
    @ PyCall C:\Users\arn203\.julia\packages\PyCall\7a7w0\src\PyCall.jl:550
  [2] pyimport_conda(modulename::String, condapkg::String, channel::String)
    @ PyCall C:\Users\arn203\.julia\packages\PyCall\7a7w0\src\PyCall.jl:708
  [3] _import_dependency(modulename::String, pkgname::String; channel::String)
    @ ArviZ C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\setup.jl:123
  [4] import_arviz()
    @ ArviZ C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\setup.jl:1
  [5] top-level scope
    @ C:\Users\arn203\.julia\packages\ArviZ\hisNy\src\ArviZ.jl:113
  [6] include
    @ .\Base.jl:418 [inlined]
  [7] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt64}}, source::Nothing)
    @ Base .\loading.jl:1318
  [8] top-level scope
    @ none:1
  [9] eval
    @ .\boot.jl:373 [inlined]
 [10] eval(x::Expr)
    @ Base.MainInclude .\client.jl:453
 [11] top-level scope
    @ none:1
ERROR: Failed to precompile ArviZ [131c737c-5715-5e2e-ad31-c244f01c1dc7] to C:\Users\arn203\.julia\compiled\v1.7\ArviZ\jl_67B1.tmp.
Stacktrace:
 [1] error(s::String)
   @ Base .\error.jl:33
 [2] compilecache(pkg::Base.PkgId, path::String, internal_stderr::IO, internal_stdout::IO, ignore_loaded_modules::Bool)      
   @ Base .\loading.jl:1466
 [3] compilecache(pkg::Base.PkgId, path::String)
   @ Base .\loading.jl:1410
 [4] _require(pkg::Base.PkgId)
   @ Base .\loading.jl:1120
 [5] require(uuidkey::Base.PkgId)
   @ Base .\loading.jl:1013
 [6] require(into::Module, mod::Symbol)
   @ Base .\loading.jl:997

julia> 
```
I was using ArviZ successfully a month or 2 ago. My laptop has updated since then. Is that likely to be linked to the cause?

Thanks

**Comment by sethaxen** [comment]
I wonder if this is related to the workaround for #188 implemented in #202. I hadn't tested that workaround on Windows.

Does the following work for you?
```julia
] add ArviZ#70735a9
```

**Comment by EvoArt** [comment]
Unfortunately not. I get the same error.

the file C:\\Users\\arn203\\.julia\\conda\\3\\lib\\site-packages\\scipy\\.libs\\libbanded5x.CBCBXIAYFUWOYCSBXUA2BT6W76FYX4C7.gfortran-win_amd64.dll

is definitely there also.

**Comment by sethaxen** [comment]
Hrm. Okay, if you don't mind, could you try a few more things?

Assuming your PyCall is configured to use Julia's Conda, try
```julia
using Conda, PyCall
Conda.pip_interop(true)
Conda.pip("uninstall -y", "scipy")
pyimport_conda("scipy", "scipy", "conda-forge")
```
What happens? If the error is the same, I have no clue what's going on. If it errors with the `scipy.fft` error in https://github.com/JuliaPy/PyCall.jl/issues/990, then you can try 

```bash
$ LD_PRELOAD=$HOME/.julia/conda/3/lib/libstdc++.so julia
```
or whatever the Windows version of this is. If `pyimport_conda` doesn't error, can you try

```julia
] add ArviZ#70735a9
```
again?

**Comment by sethaxen** [comment]
Hi @EvoArt, did you have any luck with my last suggestion?

**Comment by EvoArt** [comment]
Unfortunately not. I'm now getting the same error , but on this line
```
julia> pyimport_conda("scipy", "scipy", "conda-forge")
[ Info: Installing scipy via the Conda scipy package...
[ Info: Running `conda config --add channels conda-forge --file 'C:\Users\arn203\.julia\conda\3\condarc-julia.yml' --force` in root environment
Warning: 'conda-forge' already in 'channels' list, moving to the top
[ Info: Running `conda install -y scipy` in root environment
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\Users\arn203\.julia\conda\3

  added / updated specs:
    - scipy


The following packages will be SUPERSEDED by a higher-priority channel:

  scipy                       pypi/pypi::scipy-1.7.3-pypi_0 --> conda-forge/win-64::scipy-1.7.3-py38ha1292f7_0


Preparing transaction: done
Verifying transaction: done
Executing transaction: done
ERROR: PyError (PyImport_ImportModule) <class 'FileNotFoundError'>
FileNotFoundError("Could not find module 'C:\\Users\\arn203\\.julia\\conda\\3\\lib\\site-packages\\scipy\\.libs\\libbanded5x.CBCBXIAYFUWOYCSBXUA2BT6W76FYX4C7.gfortran-win_amd64.dll' (or one of its dependencies). Try 
using the full path with constructor syntax.")
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\__init__.py", line 136, in <module>
    from . import _distributor_init
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\_distributor_init.py", line 59, in <module>  
    WinDLL(os.path.abspath(filename))
  File "C:\Users\arn203\.julia\conda\3\lib\ctypes\__init__.py", line 381, in __init__
    self._handle = _dlopen(self._name, mode)

Stacktrace:
 [1] pyimport(name::String)
   @ PyCall C:\Users\arn203\.julia\packages\PyCall\7a7w0\src\PyCall.jl:550
 [2] pyimport_conda(modulename::String, condapkg::String, channel::String)
   @ PyCall C:\Users\arn203\.julia\packages\PyCall\7a7w0\src\PyCall.jl:714
 [3] top-level scope
   @ REPL[8]:1

caused by: PyError (PyImport_ImportModule) <class 'FileNotFoundError'>
FileNotFoundError("Could not find module 'C:\\Users\\arn203\\.julia\\conda\\3\\lib\\site-packages\\scipy\\.libs\\libbanded5x.CBCBXIAYFUWOYCSBXUA2BT6W76FYX4C7.gfortran-win_amd64.dll' (or one of its dependencies). Try using the full path with constructor syntax.")
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\__init__.py", line 136, in <module>
    from . import _distributor_init
  File "C:\Users\arn203\.julia\conda\3\lib\site-packages\scipy\_distributor_init.py", line 59, in <module>
    WinDLL(os.path.abspath(filename))
  File "C:\Users\arn203\.julia\conda\3\lib\ctypes\__init__.py", line 381, in __init__
    self._handle = _dlopen(self._name, mode)

Stacktrace:
 [1] pyimport(name::String)
   @ PyCall C:\Users\arn203\.julia\packages\PyCall\7a7w0\src\PyCall.jl:550
 [2] pyimport_conda(modulename::String, condapkg::String, channel::String)
   @ PyCall C:\Users\arn203\.julia\packages\PyCall\7a7w0\src\PyCall.jl:708
 [3] top-level scope
   @ REPL[8]:1
```
It's the same file `FileNotFoundError("Could not find module 'C:\\Users\\arn203\\.julia\\conda\\3\\lib\\site-packages\\scipy\\.libs\\libbanded5x.CBCBXIAYFUWOYCSBXUA2BT6W76FYX4C7.gfortran-win_amd64.dll' (or one of its dependencies). `  that cannot be found (but does exist in that location). We can safely say this is not an arviz problem. More likely something to do with using an old uni issue laptop, with all its university standard configurations.

**Comment by sethaxen** [comment]
Hi @EvoArt we now explicitly test that ArviZ installs correctly on windows, so can you check if ArviZ v0.6.6 works for you?

**Comment by sethaxen** [comment]
Fixed by #290

**Comment by EvoArt** [comment]
This was a VERY nice fix 😅🙏
